src/components/DeviceMonitNew/index.js

```diff
@@ -134,11 +134,12 @@ class DeviceMonit extends PureComponent<Props, State> {
   }
 
   render() {
+    const { coinType, account, devices, deviceSelected, render } = this.props
     const { status } = this.state
-    const { devices, deviceSelected, render } = this.props
 
     if (render) {
       return render({
+        coinType: (account && account.coinType) || coinType,
```

Comment:
- according to type, coinType is always here. not a `?number`
- coinType === 0 is actually legit, which is bitcoin. in that case `0 || coinType` might fallback to undefined if coinType not in props ?

Reply:
Yep, but `account` in this context are not always here

Reply:
In this context, you have account with coinType (for Send and Receive flow), or just coinType (for AddAccount flow)

Reply:
ok so `account ? account.coinType : cointType` maybe?